tests/helpers/utils.ts

```diff
@@ -51,7 +52,9 @@ export const setUpServerAndDb = async ({
     ...blockchainSettings,
   })
   await delay(5 * 1000)
-  return { db, server }
+  const rabbitMQ = await RabbitMQ(process.env.RABBITMQ_URL)
```

Comment:
What do you say if we change the name `setUpServerAndDb` to `setUpApplication` ? (in another PR)

Reply:
yes, that would be a good idea.